docs/tutorials/timeslices.md

```diff
@@ -0,0 +1,258 @@
+<!--
+SPDX-FileCopyrightText: Copyright (c) 2024-2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# Time Slicing for Performance Analysis
+
+Time slicing allows you to analyze performance metrics across sequential time windows during a benchmark run. This feature provides visibility into performance trends, degradation patterns, and system behavior over time.
+
+## Overview
+
+Time slicing divides your benchmark into equal duration segments, computing metrics independently for each segment. This enables:
+
+- **Performance Trend Analysis**: Identify if performance degrades, improves, or stabilizes over time
+- **Warm-up Detection**: Distinguish initial cold-start behavior from steady-state performance
+- **Resource Exhaustion**: Spot gradual performance degradation due to memory leaks or resource pressure
+- **Load Pattern Impact**: Understand how different phases of load affect system performance
+- **Time-series Visualization**: Export data suitable for plotting performance trends
+
+## Core Parameters
+
+### Slice Duration
+- `--slice-duration MILLISECONDS`: Duration of each time slice
+- Recommended to be used with `--benchmark-duration`
+- Creates non-overlapping sequential time windows
+- Example: 10-second benchmark with 1000-millisecond slices creates 10 time windows
+
+### Benchmark Duration
+- `--benchmark-duration SECONDS`: Total benchmark duration
+- Must be greater than `--slice-duration`
+- Determines how many slices will be created
+
+## Basic Time Slicing
+
+### Setting Up the Server
+
+```bash
+# Start vLLM server for time slicing demonstration
+docker pull vllm/vllm-openai:latest
+docker run --gpus all -p 8000:8000 vllm/vllm-openai:latest \
+  --model Qwen/Qwen3-0.6B \
+  --host 0.0.0.0 --port 8000 &
+```
+
+```bash
+# Wait for server to be ready
+timeout 900 bash -c 'while [ "$(curl -s -o /dev/null -w "%{http_code}" localhost:8000/v1/chat/completions -H "Content-Type: application/json" -d "{\"model\":\"Qwen/Qwen3-0.6B\",\"messages\":[{\"role\":\"user\",\"content\":\"test\"}],\"max_tokens\":1}")" != "200" ]; do sleep 2; done' || { echo "vLLM not ready after 15min"; exit 1; }
+```
+
+### Simple Time Slicing Example
+
+Run a 60-second benchmark with 10-second slices to analyze performance trends:
+
+```bash
+aiperf profile \
+  --model Qwen/Qwen3-0.6B \
+  --benchmark-duration 60 \
+  --slice-duration 10000
+```
+
+This creates 6 time slices (0-10s, 10-20s, 20-30s, 30-40s, 40-50s, 50-60s), each with independent metrics.
```

Comment:
is it confusing to the user with the different units? maybe it should be renamed to `--slice-duration-ms`? or switch it to floating point seconds?

Reply:
I agree that it is confusing.

I like seconds for both. Changed.